### PR TITLE
Added new link for course and free programming playgrounds

### PR DESCRIPTION
**Pull request by ritikasaha2002**
## What does this PR do?
Add resource(s)

## For resources
Added new course link for Data Structure by Jenny's Lectures CS IT and also added new python compiler under free playgrounds.

### Why is this valuable (or not)?
It will serve as a beneficial course in free to learn data structure in C Programming language. Also students can run their code for free through the online python compiler of Tutorialspoint.

### How do we know it's really free?
The links provided by me are free. Both, the youtube course and the online compiler can be accessed for free and does not require signing up.

### For book lists, is it a book? For course lists, is it a course? etc.
I have added a course and a online compiler link.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Thenlie** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems. Once you have made the necessary changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -200,6 +200,7 @@
 * [Data Structure And Algorithms Course | DSA Tutorial in Hindi](https://www.youtube.com/playlist?list=PLmGElG-9wxc9Us6IK6Qy-KHlG_F3IS6Q9) - ScoreShala
 * [Data Structure Programs \| Hindi](https://www.youtube.com/playlist?list=PLsFNQxKNzefJU-Sj__mljvrmJHZVKWbEm) - Saurabh Shukla
 * [Data Structures and Algorithms Course in Hindi](https://www.youtube.com/playlist?list=PLu0W_9lII9ahIappRPN0MCAgtOu3lQjQi) - CodeWithHarry
+* [Data Structures and Algorithms](https://youtu.be/AT14lCXuMKI?si=cubF_ZiTYJ1EfXcL) - Jenny's Lectures CS IT
```
Comment (eshellman) [inline courses/free-courses-hi.md:203]:
```suggestion
* [Data Structures and Algorithms](https://www.youtube.com/playlist?list=PLdo5W4Nhv31bbKJzrsKfMpo_grxuLl8LU) - Jenny's Lectures CS IT
```

File: more/free-programming-playgrounds.md [hunk 2]
```diff
@@ -269,6 +269,7 @@
 * [Repl.it - NiceDualPoint](https://repl.it/repls/NiceDualPoint#main.py)
 * [SoloLearn](https://code.sololearn.com/python)
 * [Try Python](https://trypython.jcubic.pl) - Jakub T. Jankiewicz
+* [Tutorialspoint](https://www.tutorialspoint.com/online_python_compiler.php)
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:272]:
```suggestion
* [Online Python Compiler](https://www.tutorialspoint.com/online_python_compiler.php) - Tutorialspoint
```

File: more/free-programming-playgrounds.md [hunk 3]
```diff
@@ -271,6 +272,7 @@
 * [Try Python](https://trypython.jcubic.pl) - Jakub T. Jankiewicz
 
 
+
```
Comment (eshellman) [inline more/free-programming-playgrounds.md:275]:
```suggestion
```

**Comment by eshellman** [comment]
added in #10988

**Pull request closed without merging**